src/components/TooltipController/TooltipController.tsx

```diff
@@ -85,7 +85,7 @@ const TooltipController = ({
         setTooltipOffset(value === null ? offset : Number(value))
       },
       wrapper: (value) => {
-        setTooltipWrapper((value as WrapperType) ?? 'div')
```

Comment:
the fix.

when the component start to check anchor attributes and there's no `wrapper`, the `wrapper` state gets updated to be a `div`